beamer/cli.py

```diff
@@ -102,8 +102,8 @@ def main(
 ) -> None:
     beamer.util.setup_logging(log_level=log_level.upper(), log_json=False)
 
-    # TODO: use return value and exit if relayer is not available
-    relayer_executable_exists()
+    if get_relayer_executable() is None:
+        log.warn("No relayer available")
```

Comment:
I'm thinking here, the agent should probably refuse to start if there's no usable relayer.
Because the relayer is critical for L1 resolution and if we don't have it...
So I would actually prefer that we log an error here and immediately exit with an error code.

Reply:
I totally agree, I most mostly worried about creating more problems for the testnet deployments.